pymc3/distributions/multivariate.py

```diff
@@ -1925,6 +1924,73 @@ def _distr_parameters_for_repr(self):
         return ["mu"]
 
 
+class CARRV(RandomVariable):
+    name = "car"
+    ndim_supp = 1
+    ndims_params = [1, 2, 0, 0]
+    dtype = "floatX"
+    _print_name = ("CAR", "\\operatorname{CAR}")
+
+    def make_node(self, rng, size, dtype, mu, W, alpha, tau):
+        mu = at.as_tensor_variable(floatX(mu))
+
+        W = aesara.sparse.as_sparse_or_tensor_variable(floatX(W))
+        if not W.ndim == 2:
```

Comment:
This check seems less restrictive than the original one, and it only covers the number of dimensions. Any reason for the change? You can add a `symbolic` assert if you move these checks to the `dist` classmethod.

Reply:
@ricardoV94 , I removed symmetry check because I can not find an implementation of  ```all_close``` for aesara sparse matrices. I only found these one https://aesara.readthedocs.io/en/latest/library/tensor/basic.html#aesara.tensor.allclose for dense. 

In the discussion above (https://github.com/pymc-devs/pymc3/pull/4596#discussion_r606376156), @brandonwillard said that ```as_tensor_variable``` calls should be placed in the ```make_node``` method rather than ```dist```. And I have done that.

Reply:
> @ricardoV94 , I removed symmetry check because I can not find an implementation of `all_close` for aesara sparse matrices. I only found these one https://aesara.readthedocs.io/en/latest/library/tensor/basic.html#aesara.tensor.allclose for dense.

What happens with the sampling / logp if the matrix does not respect the old requirement? Does it fail gracefully or does it return incorrect samples / logp evaluations? If it fails gracefully it should be fine to continue without the check (but then the error message should be changed since it only checks if `W` is a matrix.)

Reply:
@ricardoV94 , If ```W``` not the symmetric matrix, wrong samples will be returned because it is not CAR distribution anymore. Looks not very good, so I will try to fix it somehow.